source/Octopus.Server.Client/Octopus.Server.Client.csproj

```diff
@@ -47,7 +47,7 @@ This package contains the non-ILmerged client library for the HTTP API in Octopu
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
     <PackageReference Include="Octodiff" Version="1.2.1" />
-    <PackageReference Include="Octopus.Server.MessageContracts.Base" Version="3.0.400" />
+    <PackageReference Include="Octopus.Server.MessageContracts.Base" Version="3.0.549" />
     <PackageReference Include="Octopus.Server.MessageContracts.Base.HttpRoutes" Version="3.0.400" />
```

Comment:
Does this needs to be bumped at the same time?

Reply:
Good question I'm going to try reverting this change and run the server tests, this might not be necessary at all